src/web/client/common/authenticationProvider.ts

```diff
@@ -7,13 +7,12 @@ import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
 nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFormat.standalone })();
 import { sendErrorTelemetry, sendInfoTelemetry } from '../telemetry/webExtensionTelemetry';
-import { pathParamToSchema, PROVIDER_ID, telemetryEventNames } from './constants';
-import PowerPlatformExtensionContextManager from "./localStore";
+import { PROVIDER_ID, telemetryEventNames } from './constants';
 
-export function getHeader(accessToken: string) {
+export function getHeader(accessToken: string, useOctetStreamContentType?: boolean) {
```

Comment:
Should we start using more standard library implementation like axios rather than building headers manually?

Reply:
Out of scope. We can consider this in  later PR.